odbc_fdw.c

```diff
@@ -1367,3 +1578,288 @@ odbcReScanForeignScan(ForeignScanState *node)
 		elog(DEBUG1, "odbcReScanForeignScan");
 	#endif
 }
+
+
+static void
+appendOption(StringInfo str, int first, const char* option_name, const char* option_value)
```

Comment:
For `first` better to use `bool` if you mean a boolean so that the header of the function speaks for itself.

(widely used in postgres source code, and also in this very file)